end-to-end-test/remote/specs/core/comparisonTab.spec.js

```diff
@@ -1,154 +1,229 @@
 
 describe('results view comparison tab venn diagram tests', function() {
     describe('create new group tests', function() {
-        before(function() {
-            goToUrlAndSetLocalStorage(
+        before(async function() {
+            // await browser.debug()
+            await goToUrlAndSetLocalStorage(
                 `${CBIOPORTAL_URL}/results/comparison?Z_SCORE_THRESHOLD=2.0&cancer_study_id=coadread_tcga_pub&cancer_study_list=coadread_tcga_pub&case_set_id=coadread_tcga_pub_nonhypermut&gene_list=KRAS%20NRAS%20BRAF&gene_set_choice=user-defined-list&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations`
             );
-            $('div[data-test="ComparisonPageOverlapTabDiv"]').waitForExist({
+            await getElement(ComparisonPageOverlapTabDiv, {
                 timeout: 20000,
             });
         });
 
-        it('create group button disabled as default', function() {
-            assert.equal($(SampleCreateGroupButton).isEnabled(), false);
-            assert.equal($(PatientCreateGroupButton).isEnabled(), false);
+        it('create group button disabled as default', async function() {
+            assert.equal(
+                await (await getElement(SampleCreateGroupButton)).isEnabled(),
+                false
+            );
+            assert.equal(
+                await (await getElement(PatientCreateGroupButton)).isEnabled(),
+                false
+            );
         });
 
-        it('select from sample venn diagram', function() {
+        it('select from sample venn diagram', async function() {
             jsApiClick('rect[data-test="sample0VennRegion"]');
-            browser.pause(100);
-            assert.equal($(SampleCreateGroupButton).isEnabled(), true);
-            assert.equal($(PatientCreateGroupButton).isEnabled(), false);
+            await browser.pause(100);
+            assert.equal(
+                await (await getElement(SampleCreateGroupButton)).isEnabled(),
+                true
+            );
+            assert.equal(
+                await (await getElement(PatientCreateGroupButton)).isEnabled(),
+                false
+            );
         });
 
-        it('click sample venn diagram create group button', function() {
-            $(SampleCreateGroupButton).click();
-            $('div.rc-tooltip-inner').waitForExist({ timeout: 20000 });
-            browser.pause(100);
-            assert.equal(
-                $('[data-test="sampleGroupNameInputField"]').isDisplayed(),
-                true,
-                'group name input exists'
+        it('click sample venn diagram create group button', async function() {
+            (await getElement(SampleCreateGroupButton)).click();
+            await getElement('div.rc-tooltip-inner', { timeout: 20000 });
+            await browser.pause(200);
+            const el = await getElement(
```

Comment:
lets try to convert these to getElementByTestHandle (whenever you see we are using data-test)